cypress/fixtures/search_page_spec/all_datasets_last_modified.json

```diff
@@ -0,0 +1,486 @@
+{
```

Comment:
I like the idea here, though I'd be inclined to distill this down to just what we need and then maybe make it more generic at that.

Reply:
Could you explain a little more?